.github/workflows/pr-realm-js.yml

```diff
@@ -293,24 +255,22 @@ jobs:
 
       - name: Restore NPM cache
         id: npm-cache
-        uses: actions/cache@v3
+        uses: actions/cache@v4
         with:
           path: ${{ steps.npm-cache-dir.outputs.dir }}
           key: ${{ runner.os }}-node-${{ hashFiles('package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
 
       - name: Restore Wireit cache
-        uses: google/wireit@setup-github-actions-caching/v1
+        uses: takameyer/wireit@wireit-caching-gha
 
       - name: Restore React Native cache
-        if: ${{ matrix.variant.environment == 'react-native' }}
+        if: ${{ matrix.variant.os == 'ios' }}
         uses: actions/cache@v3
         with:
           path: '**/Pods'
-          key: ${{ runner.os }}-${{matrix.variant.environment}}-${{ hashFiles('**/Podfile.lock', './src/**', './vendor/**') }}
-          restore-keys: |
-            ${{ runner.os }}-${{matrix.variant.environment}}-
+          key: ${{ runner.os }}-${{matrix.variant.environment}}-${{ hashFiles('**/Podfile.lock', './packages/realm/binding/src/**', './packages/realm/bindgen/vendor/**', './packages/realm/RealmJS.podspec', './packages/realm/scripts/*') }}
```

Comment:
Currently with one vendor it doesn't really matter, but would it be preferable to use `'./packages/realm/bindgen/vendor/realm-core/**'` instead of `../vendor/`?